src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -125,6 +140,34 @@ private void checkMapping(SinkRecord record) {
     }
   }
 
+  private String getServerVersion() {
+    ConfigCallbackHandler configCallbackHandler = new ConfigCallbackHandler(config);
+    this.highLevelClient = highLevelClient != null ? highLevelClient : new RestHighLevelClient(
+        RestClient
+            .builder(
+                config.connectionUrls()
+                    .stream()
+                    .map(HttpHost::create)
+                    .collect(Collectors.toList())
+                    .toArray(new HttpHost[config.connectionUrls().size()])
+            )
+            .setHttpClientConfigCallback(configCallbackHandler)
+            .setRequestConfigCallback(configCallbackHandler)
+    );
+    MainResponse response;
+    String esVersionNumber = "Unknown";
+    try {
+      response = highLevelClient.info(RequestOptions.DEFAULT);
+      esVersionNumber = response.getVersion().toString();
+    } catch (Exception e) {
+      log.info("Failed to get ES server version", e);
```

Comment:
log.error

Reply:
error is quite strict for this soft information. 
`log.warn` seems like a good compromise

Reply:
Also, if we use the high level client only to get the version, should we close it (and discard it) here?
What's the life cycle of this client? It might utilized threads, etc.

Reply:
This is even better idea.  Agree on removing the instance level field and closing it here.